app/src/components/requestLogs/LogFilters/LogFilter.tsx

```diff
@@ -0,0 +1,49 @@
+import { useCallback, useState } from "react";
+import { HStack, IconButton, Input } from "@chakra-ui/react";
+import { BsTrash } from "react-icons/bs";
+
+import { type LogFilter } from "~/state/logFiltersSlice";
+import { useAppStore } from "~/state/store";
+import { debounce } from "lodash-es";
+import SelectFieldDropdown from "./SelectFieldDropdown";
+import SelectComparatorDropdown from "./SelectComparatorDropdown";
+
+const LogFilter = ({ filter, index }: { filter: LogFilter; index: number }) => {
+  const updateFilter = useAppStore((s) => s.logFilters.updateFilter);
+  const deleteFilter = useAppStore((s) => s.logFilters.deleteFilter);
+
+  const [editedValue, setEditedValue] = useState("");
+
+  const debouncedUpdateFilter = useCallback(
+    debounce(
```

Comment:
Debouncing to save our poor database

Reply:
lol debouncing sounds good, but maybe we want >200ms? We might actually need to wait until someone clicks "save filters" or something. This is likely going to slam us once a user has have a non-trivial number of logs (more than a few hundred, even).